src/eosjs-numeric.ts

```diff
@@ -327,6 +327,24 @@ export function stringToPublicKey(s: string): Key {
     }
 }
 
+export function publicKeyToLegacyString(key: Key) {
```

Comment:
This could be named something specific to K1 if we're only supporting conversion between K1 and public keys with this, because hiding a throw for other key types is confusing.

Reply:
Since it's available through `toLegacyString()` in `PublicKey`, the throw could be moved there.  However, all `PublicKey` instances will be able to call `toLegacyString()` and changing it to something like `toK1LegacyString()` might also be confusing.  It would imply you could convert anything to K1 Legacy. A hidden throw is still required.